src/classes/RD2_OpportunityEvaluation_BATCH.cls

```diff
@@ -129,11 +157,18 @@ public without sharing class RD2_OpportunityEvaluation_BATCH
     * @description Batch Start Method
     */
     public Database.QueryLocator start(Database.BatchableContext context) {
-        if (RD2_EnablementService.isRecurringDonations2Enabled) {
-            return Database.getQueryLocator(getRDBatchQuery());
-        } else {
+        
+        if (!RD2_EnablementService.isRecurringDonations2Enabled) {
```

Comment:
@rponti-sforg , can you please also try to ensure that RD2 batch throws an error when RD2 is not enabled , ie this check continues to work as expected.